src/aggregator/swap-container/index.tsx

```diff
@@ -129,14 +133,31 @@ const SwapContainer = () => {
     if (to) {
       replaceHistory(`/swap/${currentNetwork.chainId}/${to.address || ''}/${from?.address || ''}`);
     }
+    trackEvent('Aggregator - Toggle from/to', { fromAddress: from?.address, toAddress: to?.address });
   };
 
   const onSetSelectedRoute = (newRoute: SwapOption) => {
     dispatch(setSelectedRoute(newRoute));
+    trackEvent('Aggregator - Change selected route');
```

Comment:
Maybe a 'from source' - 'to source' might be cool here!

Reply:
Sure